internal/services/cosmos/cosmosdb_cassandra_datacenter_resource.go

```diff
@@ -109,6 +109,15 @@ func resourceCassandraDatacenter() *pluginsdk.Resource {
 				Optional: true,
 				Default:  true,
 			},
+
+			"seed_node_ip_addresses": {
+				Type:     pluginsdk.TypeList,
+				Computed: true,
+				Elem: &pluginsdk.Schema{
+					Type:         pluginsdk.TypeString,
+					ValidateFunc: validation.IsIPv4Address,
```

Comment:
since this is computed we don't need to validate this

Reply:
Updated